sysdeps/managarm/x86_64/crt-src/Scrt1.s

```diff
@@ -0,0 +1,5 @@
+.section .text
```

Comment:
The filename ends in `.s` (lower case) while it should be `.S`. This results in a build failure on my local machine.